src/Native/Unix/System.Security.Cryptography.Native/pal_bio.c

```diff
@@ -6,49 +6,49 @@
 
 #include <assert.h>
 
-extern "C" BIO* CryptoNative_CreateMemoryBio()
+BIO* CryptoNative_CreateMemoryBio()
```

Comment:
Add `void` inside the parentheses for consistency with other places where you have added the `void`?

Reply:
Compiler only requires it in first declaration and subsequent forward declaration. It hasn't been done in previous PRs, and not a compile time error, should we skip it for now?

Reply:
Ok, we can skip it.